src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/TestsResources.xaml

```diff
@@ -622,4 +623,334 @@
+								</VisualState>
+
+								<VisualState x:Name="Closed">
+									<Storyboard>
+										<SplitCloseThemeAnimation ClosedTargetName="ContentPresenter"
+																	  ContentTranslationOffset="40"
+																	  ContentTranslationDirection="{Binding TemplateSettings.SelectedItemDirection, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+																	  ContentTargetName="ScrollViewer"
+																	  ClosedLength="{Binding TemplateSettings.DropDownClosedHeight, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+																	  OffsetFromCenter="{Binding TemplateSettings.DropDownOffset, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+																	  OpenedTargetName="PopupBorder"
+																	  OpenedLength="{Binding TemplateSettings.DropDownOpenedHeight, RelativeSource={RelativeSource Mode=TemplatedParent}}" />
+									</Storyboard>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="Auto" />
+							<ColumnDefinition Width="*" />
+						</Grid.ColumnDefinitions>
+
+						<!-- Pressed Background -->
+						<Border x:Name="PressedBackground"
+									Background="{ThemeResource ComboBoxPressedHighlightThemeBrush}"
+									Opacity="0"
+									Grid.ColumnSpan="2" />
+
+						<!-- Title -->
+						<TextBlock x:Name="TitleElement"
+									   Text="{TemplateBinding Text}"
+									   Style="{StaticResource Subtitle1}"
+									   Foreground="{StaticResource MaterialOnBackgroundBrush}"
+									   VerticalAlignment="Center"
+									   HorizontalAlignment="Left"
+									   IsHitTestVisible="False"
+									   MaxLines="1" />
+
+						<!-- PlaceholderElement (showing placeholder text when no selection is done) -->
+						<TextBlock x:Name="PlaceholderElement"
+									   Grid.Column="1"
+									   Text="{TemplateBinding PlaceholderText}"
+									   Style="{StaticResource Body2}"
+									   Foreground="{StaticResource MaterialOnBackgroundMediumBrush}"
+									   VerticalAlignment="Center"
+									   HorizontalAlignment="Right"
+									   IsHitTestVisible="False"
+									   MaxLines="1"
+									   Visibility="{Binding SelectedIndex, RelativeSource={RelativeSource TemplatedParent},Converter={StaticResource SelectedIndexToPlaceholderVisibility}}" />
+
+						<!-- ContentPresenter (showing the selected value) -->
+						<ContentPresenter x:Name="ContentPresenter"
+											  Grid.Column="1"
+											  VerticalAlignment="Center"
+											  HorizontalAlignment="Right" />
+
+						<!-- Popup -->
+						<Popup x:Name="Popup">
+							<Border x:Name="PopupBorder"
+										CornerRadius="{StaticResource ComboBoxRadius}"
+										Background="{StaticResource MaterialSurfaceBrush}">
+								<!-- Workaround for Issue #213 where android PopupBorder doesn't get redrawn properly, adds Background, CornerRadius and Border to ScrollViewer  -->
```

Comment:
Same here, would be best to have the link to the issue (instead of just the number)